### PR TITLE
Update queen-attack test expectations

### PR DESCRIPTION
**Pull request by IanWhitney**
Fixing my copy & paste errors that @petertseng noticed in https://github.com/exercism/x-common/pull/211

### CODE REVIEW

**Comment by kytrinyx** [comment]
This looks good to me. Would you mind updating the commit message to have `queen-attack` in it? That helps immeasurably when digging into the history.

File: queen-attack.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
       "black_queen": {
         "position": "(1,1)"
       },
-      "expected": false
+      "expected": true
```
Comment (petertseng) [inline queen-attack.json]: I believe the coordinates will need updating here if we want this to be true - (2, 4) can't attack (1, 1)

File: queen-attack.json [hunk 2]
```diff
@@ -77,7 +77,7 @@
       "black_queen": {
         "position": "(5,5)"
       },
-      "expected": false
+      "expected": true
```
Comment (petertseng) [inline queen-attack.json]: Coordinates probably need to be changed here too - (2, 4) can't attack (5, 5)

**Comment by IanWhitney** [comment]
You'd think it would be simple to write a dang json file. Apparently not. Fixing those two errors that @petertseng spotted.

**Comment by IanWhitney** [comment]
Ok. All diagonal tests now look good to me.

**Comment by petertseng** [comment]
Could you check real quick whether the change got pushed right? Still looks the same to me

**Comment by IanWhitney** [comment]
Add Git to the things I can't get right today.

**Comment by kytrinyx** [comment]
@IanWhitney -- would you mind tweaking the commit subject line? It sounds like you're having a chaotic day, and I'm happy to amend the commit message if you've got too much stuff going on. Just say the word.

**Comment by petertseng** [comment]
OK from me on the json content, so it'll be ready to go when getting the commit message sorted.

**Comment by IanWhitney** [comment]
Commit message changed. It's not that today was busy, but I'm traveling tomorrow. After that I'll be back at work and without these nice 3-4 hour stretches to think about test ordering, etc.

**Comment by kytrinyx** [comment]
Thanks! Good luck with travel (I know how that can go!)

**Pull request merged**